common/src/main/java/com/microsoft/identity/common/nativeauth/internal/commands/SignUpStartCommand.kt

```diff
@@ -23,17 +23,17 @@
 package com.microsoft.identity.common.nativeauth.internal.commands
 
 import com.microsoft.identity.common.nativeauth.internal.controllers.NativeAuthMsalController
-import com.microsoft.identity.common.java.nativeauth.commands.parameters.BaseSignUpStartCommandParameters
 import com.microsoft.identity.common.java.nativeauth.controllers.results.SignUpStartCommandResult
 import com.microsoft.identity.common.java.logging.LogSession
 import com.microsoft.identity.common.java.logging.Logger
+import com.microsoft.identity.common.java.nativeauth.commands.parameters.SignUpStartCommandParameters
 
 /**
  * Command class to call controllers to start the sign up flow.
  * {@see com.microsoft.identity.common.java.controllers.CommandDispatcher}.
  */
 class SignUpStartCommand(
-    private val parameters: BaseSignUpStartCommandParameters,
+    private val parameters: SignUpStartCommandParameters,
```

Comment:
Do we still need this base class, now that there is only 1 child of it?

Reply:
The base class has been deleted.